### PR TITLE
Mvh refactor

### PR DESCRIPTION
**Pull request by mikez321**
Reorganize tests and stattracker.  Create two it4 methods and tests.

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -137,43 +137,6 @@ def test_winningest_team
...
-
-  # def test_it_can_return_total_games_by_season
-  #   expected = {
-  #     "20122013"=>5,
-  #     "20152016"=>4
-  #     }
-  #   assert_equal expected, @stat_tracker.total_games_by_season
-  # end
-  #
   def test_it_can_count_total_games_by_team
```
Comment (aperezsantos) [inline test/stat_tracker_test.rb:140]: Perhaps we can format this differently.

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -266,50 +229,21 @@ def test_it_can_display_home_or_away_ties_by_team
...
+####### It4 Methods #############
+def test_team_with_the_most_tackles_in_a_season
+  assert_equal "FC Dallas", @stat_tracker.most_tackles("20122013")
+  assert_equal "Houston Dynamo",@stat_tracker.most_tackles("20152016")
+end
+
+def test_team_with_the_fewest_tackles_in_a_season
+  assert_equal "Philadelphia Union", @stat_tracker.fewest_tackles("20122013")
+  assert_equal "FC Cincinnati", @stat_tracker.fewest_tackles("20152016")
+
```
Comment (aperezsantos) [inline test/stat_tracker_test.rb:241]: Extra space here

**Review by aperezsantos — APPROVED** [review]

File: test/stat_tracker_test.rb (hunk 1 above)
Comment (colintalex) [reply test/stat_tracker_test.rb:140]: Potentially, you're referring to the hash bracket?

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -150,7 +150,6 @@ def worst_defense
 
   def lowest_scoring_visitor
     #team with lowest totals when playing as visitor
-    require "pry"; binding.pry
```
Comment (colintalex) [inline lib/stat_tracker.rb:153]: I know who left that there...... Woops

**Review by colintalex — APPROVED** [review]

**Pull request merged**